src/pyromaths/ex/cinquiemes/aires.py

```diff
@@ -40,13 +40,13 @@ def exo_aire_diques():
     rayon2 = int(1.5 * rayon1)
     i = random.randrange(2)
     if i == 0:
-        donnees = ('rayons', rayon1, rayon2)
+        donnees = (_('rayons'), rayon1, rayon2)
     else:
-        donnees = (u'diamètres', 2 * rayon1, 2 * rayon2)
+        donnees = (_(u'diamètres'), 2 * rayon1, 2 * rayon2)
     difference_des_carres = rayon2 ** 2 - rayon1 ** 2
     aire_arrondie = arrondir(str(3.14 * difference_des_carres))
     enonce = \
-        u"""\\begin{minipage}{4cm}
+        _(u"""\\begin{minipage}{4cm}
```

Comment:
Maybe this string should be splitted: the beginning (containing all the LaTeX commands) should be less as is, while the last part (containing the French text) should be translated.